src/dual.jl

```diff
@@ -302,7 +302,7 @@ end
 # Predicates #
 #------------#
 
-isconstant(d::Dual) = iszero(partials(d))
+isconstant(d::Dual) = Base.iszero(partials(d))
```

Comment:
```suggestion
isconstant(d::Dual) = iszero(partials(d))
```
The qualification from Base shouldn't actually be necessary since you're changing the `iszero` definition here to extend the one in Base. Or am I missing something?

Reply:
Indeed. It no longer needs the qualification. I've changed it.